<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 88, 88);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{	
    border-style :outset;
	border-color:#ffffff;
	border-width: 1px;
	border-radius: 3px;
}</string>
   </property>
   <widget class="QLabel" name="logo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>221</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    border-style :outset;
	border-color: rgb(26, 120, 179);
	border-width: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Account">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
 border: 1px solid #A0A0A0;
 border-radius: 3px;
color: rgb(222, 222, 222);
 selection-color: #F2F2F2;
}
QLineEdit:hover{
 border: 1px solid rgb(19, 23, 255);.
 border-radius: 3px;
 color: #F2F2F2;
 selection-color: #F2F2F2
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Myself Account</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
 border: 1px solid #A0A0A0;
 border-radius: 3px;
color: rgb(222, 222, 222);
 selection-color: #F2F2F2;
}
QLineEdit:hover{
 border: 1px solid rgb(19, 23, 255);.
 border-radius: 3px;
 color: #F2F2F2;
 selection-color: #F2F2F2
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loginAccount">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style :outset;
	border-color: rgb(26, 120, 179);
	border-width: 2px;
	border-radius: 10px;
	color: rgb(250, 250, 250);
	background-color: rgb(102, 102, 102);
}
QPushButton:hover{
	border-style :outset;
	border-color: rgb(26, 120, 179);
	border-width: 2px;
	border-radius: 10px;
	color: rgb(102, 102, 102);
	background-color: rgb(250, 250, 250);
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loginVisitor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-style :outset;
	border-color: rgb(26, 120, 179);
	border-width: 2px;
	border-radius: 10px;
	color: rgb(250, 250, 250);
	background-color: rgb(102, 102, 102);
}
QPushButton:hover{
	border-style :outset;
	border-color: rgb(26, 120, 179);
	border-width: 2px;
	border-radius: 10px;
	color: rgb(102, 102, 102);
	background-color: rgb(250, 250, 250);
}</string>
    </property>
    <property name="text">
     <string>Visitor</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginSuggestion">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
 border: 0px solid #A0A0A0;
 border-radius: 0px;
	color: rgb(223, 79, 53);
}
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
